DESCRIPTION.md

```diff
@@ -8,7 +8,7 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
-- v3.9.1(TBD)
+- v3.9.1(April 22,2024)
 
   - Fixed 400 error when connecting to a China endpoint.
```

Comment:
Fixed an issue that caused an HTTP 400 error when connecting to a China endpoint.